Lavanya/HTMLassignment3.html

```diff
@@ -0,0 +1,42 @@
+<html>
+<head>
+	<title>HTMLassignment3</title>
+	<meta charset="UTF-8">
+	<style>
+		label {
+  		cursor: default;
+		}
+	</style>
+</head>
+<body>
+
+	<h2>Register</h2>
+
+	<form id="RegistrationForm">
+		<label for="First Name">First Name</label>
+		<input type="text" name="First Name"required/>
+		<label for="Last Name">Last Name</label>
+		<input type="text" name="lastname"required>

```

Comment:
Refer this: [HTML <input> placeholder Attribute - w3schools.com](https://www.w3schools.com/tags/att_input_placeholder.asp)